prow/github/client.go

```diff
@@ -2186,6 +2185,78 @@ func (c *Client) ListTeamMembers(id int, role string) ([]TeamMember, error) {
 	return teamMembers, nil
 }
 
+// ListTeamRepos gets a list of team repos for the given team id
+//
+// https://developer.github.com/v3/teams/#list-team-repos
+func (c *Client) ListTeamRepos(id int) ([]Repo, error) {
```

Comment:
I'd be inclined to put the client changes in their own commit

Reply:
done